Client/game_sa/CDamageManagerSA.cpp

```diff
@@ -142,11 +142,13 @@ void CDamageManagerSA::SetPanelStatus(BYTE bPanel, BYTE bPanelStatus)
                 // Call CAutomobile::SetPanelDamage to update the vehicle
                 dwFunction = 0x6B1480;
                 dwThis = (DWORD)internalEntityInterface;
-                bool bUnknown = false;
+                bool windscreenShatter = bPanel == ePanels::WINDSCREEN_PANEL && breakGlass;
+                bool quiet = !spawnFlyingComponent;
                 _asm
                 {
                     mov     ecx, dwThis
-                    push    bUnknown
+                    push    quiet
+                    push    windscreenShatter
```

Comment:
Do you have stack issues after this adding a new argument?

Reply:
Nope